### DISCUSSION TITLE
Not working on Firefox.

### DISCUSSION THREAD

**Discussion by Jason5Lee** [static-web-server/static-web-server#120] [Q&A]
I just download the 2.9.0 msvc version. Works fine on Chrome. But when I open the same link in Firefox it just stuck at connection. I'm not sure which part goes wrong. Has anyone encountered this issue before? Any tip to fix it?

**Comment by joseluisq** [comment]
Can you please provide more context on how are you running SWS?

**Reply by Jason5Lee** [reply]
Command: `static-web-server --root . --port 8080`
Firefox version: Version 102.0

**Reply by joseluisq** [reply]
<img src="https://user-images.githubusercontent.com/1700322/176847177-6e224f03-ce57-4d7c-b6f6-8a194cab53d1.PNG" width="400">
